File: app/src/test/java/com/smlnskgmail/jaman/remotetemperaturecontrol/TestSuite.java
package com.smlnskgmail.jaman.remotetemperaturecontrol;

import com.smlnskgmail.jaman.remotetemperaturecontrol.datasend.DataSendTest;
import com.smlnskgmail.jaman.remotetemperaturecontrol.datavalidation.DataValidationTest;
import org.junit.runner.RunWith;
import org.junit.runners.Suite;

@RunWith(Suite.class)
@Suite.SuiteClasses({
        DataValidationTest.class,
        DataSendTest.class
})
public class TestSuite {}

File: app/src/test/java/com/smlnskgmail/jaman/remotetemperaturecontrol/TestSuite.kt
package com.smlnskgmail.jaman.remotetemperaturecontrol

import com.smlnskgmail.jaman.remotetemperaturecontrol.api.MonitorDataExtractorTest
import com.smlnskgmail.jaman.remotetemperaturecontrol.api.MonitorDataSendTest
import com.smlnskgmail.jaman.remotetemperaturecontrol.api.entities.BtDeviceTest
import org.junit.runner.RunWith
import org.junit.runners.Suite

@RunWith(Suite::class)
@Suite.SuiteClasses(
    MonitorDataExtractorTest::class,
    MonitorDataSendTest::class,
    BtDeviceTest::class
)
class TestSuite
